public/app/features/plugins/admin/pages/Browse.test.tsx

```diff
@@ -39,46 +39,88 @@ function setup(path = '/plugins'): RenderResult {
 
 describe('Browse list of plugins', () => {
   it('should list installed plugins by default', async () => {
-    const { getByText, queryByText } = setup('/plugins');
+    const { queryByText } = setup('/plugins');
 
-    await waitFor(() => getByText('Installed'));
+    await waitFor(() => queryByText('Installed'));
 
     for (const plugin of installed) {
-      expect(getByText(plugin.name)).toBeInTheDocument();
+      expect(queryByText(plugin.name)).toBeInTheDocument();
     }
 
     for (const plugin of remote) {
       expect(queryByText(plugin.name)).toBeNull();
     }
   });
 
-  it('should list all plugins when filtering by all', async () => {
-    const plugins = [...installed, ...remote];
-    const { getByText } = setup('/plugins?filterBy=all');
+  it('should list all plugins (except core plugins) when filtering by all', async () => {
```

Comment:
**Question:** why are we hiding the core plugins in this case?

Only asking out of curiosity. (As I saw that we are showing them if we filter for installed plugins, however when we want to see "all" then we only show the "3rd party" installed ones)

Reply:
Thanks for the feedback. It's an attempt to reduce the number of plugins shown under `all` as core plugins are always installed but you're right it's rather misleading. Either we remove this functionality or we rename the `all` filter to better represent the `marketplace` plugins. Will discuss it with the team and update accordingly.